### DISCUSSION TITLE
How long does it take for renovate to update dependencies?

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [renovatebot/renovate#9089] [Request Help]
Several days a go, I migrated several of my repositories from Dependabot to Renovate. However, I've noticed that Renovate hasn't updated dependencies in several days—including ones for which I know there are newer versions on NPM. Looking at the renovate app dashboard, the Renovate Bot is running several times each day, it's just not doing anything. And this isn't an issue of maxed out PR requests: there are none currently.

Here's the renovate.json for one such repo in case it matters.

```json
{
    "extends": [
        "config:base",
        ":preserveSemverRanges"
    ],
    "automerge": true,
    "automergeType": "branch",
    "lockFileMaintenance": {
        "enabled": true
    },
    "bumpVersion": "patch",
    "schedule": [
        "after 10pm every weekday",
        "before 5am every weekday",
        "every weekend"
    ]
}
```

Does it usually take a while for Renovate to update dependencies? What can I change to make the updates happen faster?

_P.S. In case the issue was the ":preserveSemverRangers" flag, I just deleted it. So perhaps that'll help? Though Dependabot was able to preserve semver while still updating daily._

**Comment by viceice** [comment]
I'm pretty sure you are using semver ranges, so renovate would nothing if the range satisfies the update.

Please provide more details or a minimal reproduction repo.

**Comment by rarkins** [comment]
You probably need `rangeStrategy=update-lockfile`. It's set to `rangeStrategy=replace` by default to keep the number of PRs down by default. Please read about it in the documentation.

**Reply by Nick-Mazuk** [reply]
Thanks, @rarkins. Updated the repo to reflect that. I'll give an update tomorrow if it's working now (once the "schedule" allows Renovate to update overnight).
